packages/webapp/pages/register.tsx

```diff
@@ -40,11 +50,16 @@ export default function Register(): ReactElement {
       <ResponsivePageContainer className="mx-auto">
         {user && (
           <>
-            <ProfileHeading>Set up your profile</ProfileHeading>
+            <ProfileHeading>
+              {getSignupModalFeatureValue(Features.SignupModalTitleCopy)}
+            </ProfileHeading>
             <h2 className="self-start my-2 text-theme-label-tertiary typo-callout">
               Please fill in your details below
             </h2>
-            <EditImageWithJoinedDate user={user} />
+            {!parseInt(
+              getSignupModalFeatureValue(Features.HideSignupModalProfileImage),
+              10,
+            ) && <EditImageWithJoinedDate user={user} />}
```

Comment:
For this one it would be better to use `isFeaturedEnabled` 

It worked like this:

```js
{!isFeaturedEnabled(Features.HideSignupModalProfileImage) && <EditImageWithJoinedDate user={user} />}
```